src/content/docs/browser/new-relic-browser/guides/guide-to-core-web-vitals.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: Monitor core web vitals
+tags:
+  - Browser
+  - Browser monitoring
+  - Guides
+metaDescription: "A guide to monitor Google's core web vitals."
+---
+
+import browserWebVitalsGif from 'images/browser_screenshot-full_web-vitals-gif.gif'
+
+import browserPageUrlView from 'images/browser_screenshot-full_page-url-view.png'
+
+import browserFilteredWebVitals from 'images/browser_screenshot-full_filtered-web-vitals.png'
+
+Core web vitals are Google’s metrics to gauge overall user experience of your site, which can influence your site’s SEO rankings and give you valuable insight into how users perceive your business.  
+
+This is why it’s important to monitor your site’s core web vitals and take action on low scores - and New Relic can help you do that! 
+
+## What are core web vitals?
+
+The three core web vitals are:
+* [Largest contentful paint](https://web.dev/lcp/) - measures loading performance
```

Comment:
I think the '-' can be replaced by ':', I can't remember to see the '-' in our docs. Examples: [Transition to New Relic from Insights](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/transition-new-relic-one-insights#features), [Error tracking with errors inbox](https://docs.newrelic.com/docs/errors-inbox/errors-inbox#dates) and [in the same page](https://docs.newrelic.com/docs/errors-inbox/errors-inbox#status), or [Account settings](https://docs.newrelic.com/docs/accounts/accounts-billing/general-account-settings/intro-account-settings#user-menu)